src/main/java/com/minecolonies/colony/permissions/Permissions.java

```diff
@@ -198,6 +211,24 @@ public void loadPermissions(@NotNull NBTTagCompound compound)
             }
             permissions.put(rank, flags);
         }
+
+        if (compound.hasKey(TAG_OWNER))
+        {
+            ownerName = compound.getString(TAG_OWNER);
+        }
+        if (compound.hasKey(TAG_OWNER_ID))
+        {
+            try
+            {
+                ownerUUID = UUID.fromString(compound.getString(TAG_OWNER_ID));
+            }
+            catch(IllegalArgumentException e)
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1166)